### ISSUE TITLE
Segments were erased incorrectly when clicked canvas

### ISSUE THREAD

**Issue by AkonXI** [bbc/peaks.js#279]
When "zoomview.displaying" events are triggered by clicking canvas to start playing audio somewhere, all segments in peaks.segments.getSegments()  are cleared.

**Comment by chrisn** [comment]
I'm not able to reproduce this issue, can you provide more detail? The following code works fine for me:

```javascript
peaksInstance.on('zoomview.displaying', function() {
  console.log(peaksInstance.segments.getSegments());
});
```
What version of Peaks.js are you using? Also, please note that the `zoomview.displaying` event is an implementation detail and not part of the public API, so may change or be removed in future.

**Comment by AkonXI** [comment]
Thank you for your reply. The reason for this error is that I incorrectly placed the create instance code in other listening methods, so every time I trigger the listening, I override the original instance.
```js

 this.$refs.audio.oncanplay = () => {
      Peaks.init(options, (err, peaks) => {
        this.ctx = peaks;
     });
  };
```
This listener should be triggered only once.
